site/content/tutorial/06-bindings/08-contenteditable-bindings/text.md

```diff
@@ -2,11 +2,16 @@
 title: Contenteditable bindings
 ---
 
-Elements with a `contenteditable="true"` attribute support `textContent` and `innerHTML` bindings:
+Elements with the `contenteditable` attribute support the following bindings:
+- [`innerHTML`](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)
+- [`innerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText)
+- [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent)
+
+There are slight differences between each of these, read more about them [here](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Differences_from_innerText).
 
 ```html
 <div
 	contenteditable="true"
 	bind:innerHTML={html}
 ></div>
-```
\ No newline at end of file
+```
```

Comment:
Suggestion: 
Update the tutorial so demonstrate the differences instead of just linking to documentation.

Reply:
Yes, great idea 👍